inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/context/InspectitContext.java

```diff
@@ -238,6 +236,15 @@ public void makeActive() {
         }
     }
 
+    private boolean anyDownPropagated() {
```

Comment:
rename maybe to "anyDownPropagatedDataOverridden" ?

Reply:
done